x/stakeibc/keeper/rebalance.go

```diff
@@ -38,7 +38,7 @@ func (k Keeper) RebalanceAllHostZones(ctx sdk.Context) {
 	}
 
 	for _, hostZone := range k.GetAllActiveHostZone(ctx) {
-		if dayEpoch.EpochNumber%hostZone.UnbondingPeriod != 0 {
+		if dayEpoch.EpochNumber%(hostZone.UnbondingPeriod+1) != 0 {
```

Comment:
mind just adding a comment as well so we don't forget (can either add it here, in the docstring, or in hooks)

Reply:
Great idea, done!